source/sync/go-to-production/optimize-sync-atlas-usage.txt

```diff
@@ -19,11 +19,15 @@ Overview
 --------
 
 Atlas Device Sync uses space in your app's synced Atlas cluster to store utility
-data for sync. This includes a history of changes to each realm.
-Atlas App Services minimizes that space usage via backend
-compaction, which runs automatically. You can optimize further by
-specifying a client maximum offline time, which enables advanced backend
-compaction.
+data for sync. This includes a history of changes to each realm. 
+Atlas App Services minimizes this space usage via backend
+compaction or trimming, which runs automatically. This optimization is 
+necessary because it:
```

Comment:
I think we would typically formulate this like:

>This optimization is necessary for the following reasons:
>- It ...
>- It ...
 
So that the intro text is a complete sentence and each of the items are complete sentences. See https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items